exercises/arrays/allPositive/allPositive.js

```diff
@@ -11,13 +11,22 @@
  * @returns {boolean} True if every number in the input array is positive
  *   and false otherwise.
  */
+
+//  How do I make the User input the array?
 function allPositive(array) {
-  // This is your job. :)
-}
+  let result = true;
+  for (let i = 0; i < array.length; i++){
+    if (array[i] <= 0){
+      result = false;
+      break;
+    }
+  }
+  console.log(result);
```

Comment:
Make sure you understand the difference between returning from a function and printing something out.

https://github.com/jfarmer/intro-javascript/blob/master/Functions/Returning.md

`console.log` is a function you call that displays whatever you pass it on the console.

`return` is a keyword that sends a value back to the place where the function was called. If you imagine handing me a stack of index cards and asking me to find the largest number, the return value is the post-it note I hand back with the answer written on it. You can do whatever you want with that value: print it out yourself, use it for some other purpose, pass it to another function, whatever.